test/api/logs.test.js

```diff
@@ -1551,4 +1554,56 @@ module.exports = () => {
                 });
         });
     });
+
+    describe('GET /api/logs/:logId/children', () => {
+        it('should return the correct child logs of a parent log', (done) => {
+            request(server)
+                .get(`/api/logs/${logWithChildrenId}/children`)
+                .expect(200)
+                .end((err, res) => {
+                    if (err) {
+                        done(err);
+                        return;
+                    }
+
+                    expect(res.body.data.every((log) => log.parentLogId === logWithChildrenId));
```

Comment:
You should also check the amount of items here

Reply:
done